src/riscv-integration.adoc

```diff
@@ -1051,7 +1049,8 @@ include::img/stval2reg.edn[]
 
 === Unprivileged CSRs
 
-Unlike machine and supervisor level CSRs, {cheri_base_ext_name} does not require
+In {cheri_base_ext_name} only writing the <<utidc>> register requires
+<<pcc>> to grant <<asr_perm>>, all other unprivileged CSRs do not require
```

Comment:
```suggestion
```suggestion
In {cheri_base_ext_name} the only register that requires <<asr_perm>> is <<utidc>> (for updates but not for reads), and all other unprivileged CSRs do not require
```
How about this wording?

Reply:
seems to have been updated accordingly